### PR TITLE
feat: created gwardia app.

### LINKED ISSUE #21: [Feat]: Create `Gwardia` app
### My suggestion

Create `Gwardia` app ([(tutorial)](https://docs.djangoproject.com/en/5.0/intro/tutorial01/#creating-the-polls-app)). Display simple hello world website.
Move NextMeeting model to `Gwardia/models.py`

### Possible alternatives

_No response_

### PR DESCRIPTION
**Pull request by mokoszerosy**
After a unnececerly long download of poetry I finally made it.

### CODE REVIEW

**Review by Norbiros — CHANGES_REQUESTED** [review]
Remove copy of NextMetting model in `core` app

**Review by Sanielia — APPROVED** [review]
Just do what Norbert asked to

**Comment by mokoszerosy** [comment]
@Norbiros done

**Review by Sanielia — CHANGES_REQUESTED** [review]
Add `gwardia` to installed apps in `settings.py` and makemigrations for `gwardia`

**Review by tapczanek1321 — CHANGES_REQUESTED** [review]
Apply these changes and i think it will be good :3

**Comment by mokoszerosy** [comment]
and again it's done

File: gwardia_hub/settings.py [hunk 1]
```diff
@@ -112,9 +113,7 @@
 # https://docs.djangoproject.com/en/5.0/howto/static-files/
 
 STATIC_URL = "static/"
-STATICFILES_DIRS = [
-    BASE_DIR / "static/GwardiaHub/"
-]
+STATICFILES_DIRS = [BASE_DIR / "static/GwardiaHub/"]
```
Comment (Sanielia) [inline gwardia_hub/settings.py:125]: Why did you move this to one line?

**Review by Sanielia — CHANGES_REQUESTED** [review]

File: gwardia_hub/settings.py (hunk 1 above)
Comment (SzczurekYT) [reply gwardia_hub/settings.py:125]: If it fits in one line then why not?

**Review by SzczurekYT — APPROVED** [review]
I have no idea how Django works, but generally LGTM

File: gwardia_hub/settings.py (hunk 1 above)
Comment (mokoszerosy) [reply gwardia_hub/settings.py:125]: honesly i dont remember doing that but i guess it doesnt matter

**Review by tapczanek1321 — APPROVED** [review]

File: gwardia_hub/settings.py (hunk 1 above)
Comment (Norbiros) [reply gwardia_hub/settings.py:125]: PRs and commits should change only things related to this. Probably Ruff/Pycharm formatter cleaned up this code... So the problem is with previous PR where someone didn't format it correctly.

File: core/templates/core/base.html [hunk 2]
```diff
@@ -13,30 +13,12 @@
         <link href="{% static 'fontawesomefree/css/brands.css' %}" rel="stylesheet" type="text/css">
         <link href="{% static 'fontawesomefree/css/solid.css' %}" rel="stylesheet" type="text/css">
     </head>
-    <body class="bg-main_bg">
+    <body class="bg-main_bg text-white">
         <main>
-            <canvas id="starback-bg" class="fixed -z-1"></canvas>
             <div class="relative z-1">
```
Comment (Kubaryt) [inline core/templates/core/base.html:18]: Get rid of this relative z-1

File: core/templates/core/index.html [hunk 3]
```diff
@@ -3,7 +3,9 @@
 {% block title %}Strona Główna{% endblock %}
 
 {% block content %}
-    <div class="flex items-center justify-center h-screen text-white">
+    <canvas id="starback-bg" class="fixed -z-1"></canvas>
+
+    <div class="flex items-center justify-center h-screen">
```
Comment (Kubaryt) [inline core/templates/core/index.html:8]: Add relative z-1 to this div

File: gwardia/templates/gwardia/index.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{% extends 'core/base.html' %}
```
Comment (Kubaryt) [inline gwardia/templates/gwardia/index.html:1]: `core/base.html` is just black background, so `gwardia/index.html` is just black background with text in left upper corner, did you want this to look like this??

image

**Review by Kubaryt — CHANGES_REQUESTED** [review]
So

File: core/templates/core/base.html (hunk 2 above)
Comment (Sanielia) [inline core/templates/core/base.html]: Delete this div, it's useless now

File: core/templates/core/index.html (hunk 3 above)
Comment (Sanielia) [inline core/templates/core/index.html]: Add relative z-1 to this div, so it displays above background

File: gwardia/templates/gwardia/index.html (hunk 4 above)
Comment (Sanielia) [inline gwardia/templates/gwardia/index.html:1]: `core/base.html` is just black background now, so `gwardia/index.html` is black background with text in left upper corner, did you want this to look like this??

image

**Review by Sanielia — CHANGES_REQUESTED** [review]

File: gwardia/templates/gwardia/index.html (hunk 4 above)
Comment (Norbiros) [reply gwardia/templates/gwardia/index.html:1]: Yes, because `core.html` adds a lot of important things like title, scripts (in future also open graph tags). It is just WIP, we can improve `/gwardia` in future.

File: core/templates/core/index.html (hunk 3 above)
Comment (mokoszerosy) [reply core/templates/core/index.html]: What?

Comment (Sanielia) [reply core/templates/core/index.html]: You have position: relative and something-z: 1 in css, and tailwind has classes for it, for position: relative `relative` class and for something-z: 1 `z-1` class, add them to this div

Comment (mokoszerosy) [reply core/templates/core/index.html]: And you assume i understand all that?

Comment (Sanielia) [reply core/templates/core/index.html]: You have to

Comment (mokoszerosy) [reply core/templates/core/index.html]: : (

**Review by Sanielia — APPROVED** [review]

**Review by Norbiros — APPROVED** [review]
